docs/DeveloperGuide.md

```diff
@@ -90,33 +90,42 @@ How the Manager class works:
 * The while loop only halts when isTerminated boolean becomes true. Then, the programme exits
 
 ### Helper Classes
+
 #### `Command`
 
-![CommandUML](https://raw.githubusercontent.com/AY2122s2-cs2113t-t09-3/tp/master/docs/Diagrams/CommandClassUML.png)
+![CommandUML](diagrams/CommandClassUML.png)
+
```

Comment:
I think add <br> under all the .png to ensure that the para is below

Reply:
Made adjustments in my latest commit.